src/Task/ImportRdfFixtures.php

```diff
@@ -22,7 +22,7 @@ public function main() {
       $file = str_replace('.rdf', '', $filename);
       $graph_name = 'http://' . strtolower($file);
       // Delete the graph first...
-      $this->execute("SPARQL CLEAR GRAPH <$graph_name>;");
+      $this->sparql("DEFINE sql:log-enable 3 CLEAR GRAPH <$graph_name>;");
```

Comment:
Is this defining extra log? Was it intended for debug reasons? If yes, should we remove it?

Reply:
The log here refers to the transaction log. Indeed, this is too confusing. I have added a comment to explain the reason and usage of it.